staging/src/k8s.io/apimachinery/pkg/util/clock/clock.go

```diff
@@ -228,26 +228,28 @@ func (i *IntervalClock) Since(ts time.Time) time.Duration {
 	return i.Time.Sub(ts)
 }
 
-// Unimplemented, will panic.
-// TODO: make interval clock use FakeClock so this can be implemented.
-func (*IntervalClock) After(d time.Duration) <-chan time.Time {
-	panic("IntervalClock doesn't implement After")
+//make interval clock use FakeClock so After can be implemented.
+func (i *IntervalClock) After(d time.Duration) <-chan time.Time {
+	fakeclock := NewFakeClock(i.Time)
+	return fakeclock.After(d)
```

Comment:
this doesn't make sense to me... without a handle to the newly created fake clock, what advances time? this seems like it would return a channel that would never close